docs/api/Apify.md

```diff
@@ -484,7 +484,7 @@ Returns a new object which contains information parsed from all the `APIFY_XXX`
 }
 ```
 
-For the list of the `APIFY_XXX` environment variables, see <a href="https://apify.com/docs/actor#run-env-vars" target="_blank">Actor
+For the list of the `APIFY_XXX` environment variables, see <a href="https://docs.apify.com/actor#run-env-vars" target="_blank">Actor
```

Comment:
Also for example here, it should be `https://docs.apify.com/actor/run#environment-variables`